notices.json

```diff
@@ -1,4 +1,23 @@
 [
+    {
+    "id": "june15-cloud-freemium",
+    "conditions": {
+      "audience": "all",
+      "clientType": "all",
+      "sku": "e0",
```

Comment:
@ogi-m please make sure this flag is thoroughly tested since we've changed our packaging since notices were built (hopefully the SKU names didn't change on the back-end though)